rfc-editor/rfc8794.xml

```diff
@@ -969,17 +891,17 @@ and deprecated by using the <tt>minver</tt> and <tt>maxver</tt> attributes of
 <t>Within an EBML Schema, the XPath of the <tt>@ebml</tt> attribute is <tt>/EBMLSchema/@ebml</tt>.</t>
 <t>The <tt>ebml</tt> attribute is a positive integer that specifies the
 version of the EBML Header (see <xref target="ebmlversion-element"></xref>)
-used by the EBML Schema. If the attribute is omitted, the EBML Header version
+used by the EBML Schema. If the  attribute is omitted, the EBML Header version
```

Comment:
?

Reply:
Ah good catch. I'll remove the remaining double spaces.
This XML file is the one the RFC editors untouched, it's not really up for review. But we do need to fix this as it ends up in the `.txt` version of the RFC.